### PR TITLE
Updates

### LINKED ISSUE #167: Split-Tunnel
Hello,
It seems that network manager creates a default route by default, even when the option "redirect-gateway" is not pushed by the server.
This prevents us to use eduvpn in split-tunnel configuration.

The solution would be to force the option "never-default" at "True" in the network manager config when we don't get the redirect-gateway parameter from the openvpn server.

What do you think ?

### LINKED ISSUE #417: Settings page back button
The back button in the top left should bring you back when on the settings page.

### LINKED ISSUE #418: Check if any remotes remain when forcing tcp
With the "force tcp"  setting enabled, the client should check if any `remote` lines remain. If there are none, the ovpn file is invalid and the client will hang. The client should handle this error instead and inform the user.

### PR DESCRIPTION
**Pull request by alvra**
Changes:
  * Fix initialization issue
  * Fix #417
  * Fix #418
  * Fix #167

### CODE REVIEW

File: eduvpn/ovpn.py [hunk 1]
```diff
@@ -1,24 +1,132 @@
...
-# matches for example: remote vpn.example.com 1234 udp
-# NOTE: multiline-mode does not consider \r
-#       so we must match for that explicitly.
-FIND_REMOTE_UDP_LINES = re.compile(
-    r'^(remote +\S+ +[0-9]+ +udp\r?)$',
-    re.MULTILINE
-)
+class Item:
+    def to_string(self) -> str:
+        raise NotImplementedError
```
Comment (gijzelaerr) [inline eduvpn/ovpn.py:8]: so this is not supposed to be called?

File: eduvpn/nm.py [hunk 2]
```diff
@@ -240,6 +241,25 @@ def get_vpn_status(client: 'NM.Client') -> Tuple['NM.VpnConnectionState', 'NM.Vp
         return vpns[0].get_state(), vpns[0].get_state_reason()
 
 
+def set_default_gateway(enable: bool):
+    "If True, make the VPN connection the default gateway."
+    _logger.debug(f"setting default gateway: {enable}")
+    uuid = get_uuid()
+    assert uuid is not None
+    command = [
```
Comment (gijzelaerr) [inline eduvpn/nm.py:249]: this should happen by means of the nm lib directly, not the nmcli.

you can see how nmcli does it in the C code

https://github.com/NetworkManager/NetworkManager/blob/main/src/nmcli/connections.c

all nm lib calls should have a python equivalent.

**Review by gijzelaerr — CHANGES_REQUESTED** [review]

File: eduvpn/ovpn.py (hunk 1 above)
Comment (alvra) [reply eduvpn/ovpn.py:8]: Not on this type, but it's part of the interface. The typechecker doesn't seem to care, but I like to make it explicit as described in the [docs](https://docs.python.org/3/library/exceptions.html#NotImplementedError).

File: eduvpn/nm.py (hunk 2 above)
Comment (alvra) [reply eduvpn/nm.py]: The latest version uses the NM api.

**Pull request merged**